nikola/conf.py.in

```diff
@@ -333,6 +333,11 @@ HIDDEN_TAGS = ['mathjax']
 # CATEGORY_PATH = "categories"
 # CATEGORY_PREFIX = "cat_"
 
+# See CATEGORY_PATH's "list of categories" for the default setting value. Can be overwritten
```

Comment:
In that case, both descriptions make no sense and should be rewritten.

How does Nikola behave when `CATEGORIES_INDEX_PATH != TAGS_INDEX_PATH`?